### ISSUE TITLE
traverse throws an exception on certain SQLAlchemy objects

### ISSUE THREAD

**Issue by benkuhn** [rollbar/pyrollbar#103]
I just discovered that a certain exception wasn't being reported to Rollbar because the stack trace included an object that was causing the traversal code to raise an exception, due to being mistaken for a `namedtuple` when it wasn't actually. Minimal example:

```python
Python 3.5.1 (default, Dec  7 2015, 21:59:08)
Type "copyright", "credits" or "license" for more information.

IPython 4.0.1 -- An enhanced Interactive Python.

In [1]: import rollbar

In [2]: from sqlalchemy import util as su

In [3]: kt = su.KeyedTuple([1, 2, 3], labels=["one", "two", "three"])

In [4]: rollbar.lib.traverse.traverse(kt)
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-ad1c4ec9283d> in <module>()
----> 1 rollbar.lib.traverse.traverse(kt)

/Users/ben/code/pyrollbar/rollbar/lib/traverse.py in traverse(obj, key, string_handler, tuple_handler, namedtuple_handler, list_handler, set_handler, mapping_handler, default_handler, circular_reference_handler, allowed_circular_reference_types, memo, **custom_handlers)
    120         return tuple_handler(tuple(traverse(elem, key=key + (i,), **kw) for i, elem in enumerate(obj)), key=key)
    121     elif obj_type is NAMEDTUPLE:
--> 122         return namedtuple_handler(obj._make(traverse(v, key=key + (k,), **kw) for k, v in iteritems(obj._asdict())), key=key)
    123     elif obj_type is LIST:
    124         return list_handler(list(traverse(elem, key=key + (i,), **kw) for i, elem in enumerate(obj)), key=key)

AttributeError: 'KeyedTuple' object has no attribute '_make'
```

This also applies to several other SQLAlchemy objects, notably a type named `result`, which is the one that actually caused our error in production. The problem is that the namedtuple type detection decides it's a namedtuple if it has a `_fields` member, but SQLA defines `_fields` on things that aren't namedtuples.

**Comment by Crisfole** [comment]
Hey Ben, we're looking into this! Thanks for letting us know.

**Comment by ezarowny** [comment]
Working on a PR here: https://github.com/rollbar/pyrollbar/pull/108

**Comment by ezarowny** [comment]
Closed in #108

**Comment by ezarowny** [comment]
Released on PyPI.
